buildSrc/extra.gradle.kts

```diff
@@ -1,3 +1,8 @@
+repositories {
```

Comment:
I don't think we gain much by refactoring these into the `extras` script. I'd rather we specify `google` and `mavenCentral` explicitly for each project. It's one extra line as we don't need to apply `extra.gradle.kts`.